packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -449,13 +450,16 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
       });
 
       //cancel all workflow reminders from steps that were edited
-      remindersToUpdate.forEach(async (reminder) => {
+      const pDeleteReminders = remindersToUpdate.map((reminder) => {
         if (reminder.method === WorkflowMethods.EMAIL) {
           deleteScheduledEmailReminder(reminder.id, reminder.referenceId);
         } else if (reminder.method === WorkflowMethods.SMS) {
           deleteScheduledSMSReminder(reminder.id, reminder.referenceId);
+        } else if (reminder.method === WorkflowMethods.WHATSAPP) {
+          deleteScheduledWhatsappReminder(reminder.id, reminder.referenceId);
```

Comment:
seems this was forgotten here, but similar calls found elsewhere in the file

Reply:
That's interesting. @CarinaWolli can you confirm that the flow was missing whatsapp here.

Reply:
I think there have been quite a few changes here since the last merge, probably fixed now!

Reply:
Not fixed on main, will revert it for now.

Reply:
I was also thinking that maybe classes would make sense here. We can have classes for all these channels(Whatsapp, SMS, Email) and the code here would simply do `channel.deleteReminder`. This would avoid having these if elses everywhere. This code is just too much involved with what's going on.

But classes might increase complexity in some other way.

Reply:
> That's interesting. @CarinaWolli can you confirm that the flow was missing whatsapp here.

Yes, definitely looks like this was missing here. Thanks for finding that 🙏

Reply:
Back in it goes :smile: